### ISSUE TITLE
Add an option open markdown notes always in view mode

### ISSUE THREAD

**Issue by ghost** [Automattic/simplenote-electron#2911]

**Comment by christianrauch** [comment]
I agree with this. Having notes set as markdown has usually the intention to have them rendered with some structure. Rendering them as plain text without the formatting defeats this purpose.

**Comment by hmlendea** [comment]
This is especially needed for notes that contain tables. They are very messy and often plain unreadable in plain text.

Simpler things such as lists are serviceable in plain text... but even they would look *much* better if rendered as markdown.

An option in Settings -> Display to toggle the default viewing mode would be really nice!
